tools/tpm2_loadexternal.c

```diff
@@ -295,6 +325,7 @@ static tool_rc process_inputs(ESYS_CONTEXT *ectx) {
     /*
      * 4.a Determine pHash length and alg
      */
+phash_alg: ;
```

Comment:
no semi-colon, typically I frown on using labels this way, from the error handling tss2_err to here seems awkward.